processor/src/main/java/org/mapstruct/ap/model/source/SourceMethod.java

```diff
@@ -61,10 +64,14 @@
     private final Accessibility accessibility;
     private final List<Type> exceptionTypes;
     private final MapperConfig config;
+    private final MappingOptions mappingOptions;
+    private final List<SourceMethod> prototypeMethods;
```

Comment:
What are the "prototype methods" here? All the methods declared on the config class(es)? Or just matching ones?